tests/src/BuiltIn/Commands/AbstractCommandTest.php

```diff
@@ -1,10 +1,35 @@
 <?php declare(strict_types = 1);
-
+ 
```

Comment:
Can you set up your IDE / texteditor to remove all trailing spaces on save please?